docker/images/n8n/README.md

```diff
@@ -147,37 +143,87 @@ The following environment variables support file input:
 - DB_POSTGRESDB_USER_FILE
 - DB_POSTGRESDB_SCHEMA_FILE
 
-## Example Setup with Lets Encrypt
+## Example server setups
 
-A basic step by step example setup of n8n with docker-compose and Lets Encrypt is available on the
-[Server Setup](https://docs.n8n.io/#/server-setup) page.
+Example server setups for a range of cloud providers and scenarios can be found in the [Server Setup documentation](https://docs.n8n.io/hosting/installation/server-setups/).
 
-## Updating a running docker-compose instance
+## Updating
 
-1. Pull the latest version from the registry
+Before you upgrade to the latest version make sure to check here if there are any breaking changes which may affect you: [Breaking Changes](https://github.com/n8n-io/n8n/blob/master/packages/cli/BREAKING-CHANGES.md)
 
-   `docker pull docker.n8n.io/n8nio/n8n`
+From your Docker Desktop, navigate to the Images tab and select Pull from the context menu to download the latest n8n image.
 
-2. Stop the current setup
+You can also use the command line to pull the latest, or a specific version:
+
+### Pull latest (stable) version
+
+```bash
+docker pull docker.n8n.io/n8nio/n8n
+```
 
-   `sudo docker-compose stop`
+### Pull specific version
 
-3. Delete it (will only delete the docker-containers, data is stored separately)
+```bash
+docker pull docker.n8n.io/n8nio/n8n:0.220.1
+```
 
-   `sudo docker-compose rm`
+### Pull next (unstable) version
 
-4. Then start it again
+```bash
+docker pull docker.n8n.io/n8nio/n8n:next
+```
 
-   `sudo docker-compose up -d`
+Stop the container and start it again:
 
-## Setting Timezone
+1. Get the container ID:
 
-To define the timezone n8n should use, the environment variable `GENERIC_TIMEZONE` can
-be set. One instance where this variable is implemented is in the Schedule node. Furthermore, the system's timezone can be set separately,
-which controls the output of certain scripts and commands such as `$ date`. The system timezone can be set via
-the environment variable `TZ`.
+```bash
+docker ps -a
+```
 
-Example to use the same timezone for both:
+2. Stop the container with ID container_id:
+
+```bash
+docker stop [container_id]
+```
+
+3. Remove the container with ID container_id:
```

Comment:
For peace of mind, it might be good to keep the note that this only deletes the container, not your data.